tests/system/python/api/test_audit.py

```diff
@@ -53,24 +53,63 @@ def test_get_severity(self, fledge_url):
         index = [idx['index'] for idx in log_severity]
         assert Counter([0, 1, 2, 4]) == Counter(index)
 
-    @pytest.mark.parametrize("request_params, total_count, audit_count, cat_name", [
-        ('', 11, 11, ''),
-        ('?limit=1', 11, 1, ''),
-        ('?skip=4', 11, 7, 'Installation'),
-        ('?limit=1&skip=8', 11, 1, 'SCHEDULER'),
-        ('?source=START', 1, 1, ''),
-        ('?source=CONAD', 10, 10, 'Utilities'),
-        ('?source=CONAD&limit=1', 10, 1, 'Utilities'),
-        ('?source=CONAD&skip=1', 10, 9, 'General'),
-        ('?source=CONAD&skip=6&limit=1', 10, 1, 'SMNTR'),
-        ('?severity=INFORMATION', 11, 11, ''),
-        ('?severity=failure', 0, 0, ''),
-        ('?source=CONAD&severity=failure', 0, 0, ''),
-        ('?source=START&severity=INFORMATION', 1, 1, ''),
-        ('?source=START&severity=information&limit=1', 1, 1, ''),
-        ('?source=START&severity=information&limit=1&skip=1', 1, 0, '')
+    @pytest.mark.parametrize("request_params, total_count, audit_count", [
+        ('', 11, 11),
+        ('?limit=1', 11, 1),
+        ('?skip=4', 11, 7),
+        ('?limit=1&skip=8', 11, 1),
+        ('?source=START', 1, 1),
+        ('?source=CONAD', 10, 10),
+        ('?source=CONAD&limit=1', 10, 1),
+        ('?source=CONAD&skip=1', 10, 9),
+        ('?source=CONAD&skip=6&limit=1', 10, 1),
+        ('?severity=INFORMATION', 11, 11),
+        ('?severity=failure', 0, 0),
+        ('?source=CONAD&severity=failure', 0, 0),
+        ('?source=START&severity=INFORMATION', 1, 1),
+        ('?source=START&severity=information&limit=1', 1, 1),
+        ('?source=START&severity=information&limit=1&skip=1', 1, 0)
     ])
-    def test_default_get_audit(self, fledge_url, wait_time, request_params, total_count, audit_count, cat_name):
+    def test_default_get_audit_sqlite(self, fledge_url, wait_time, request_params, total_count, audit_count, storage_plugin):
+        if storage_plugin == 'postgres':
+            pytest.skip('Test is for SQLite only. Storage category when with SQLite plugin, creates a child category `sqlite` hence one additional CONAD entry.')
+
+        if request_params == '':
+            time.sleep(wait_time)
```

Comment:
Could you please add why extra wait time is required?

Reply:
It should not be required, there is nothing to wait after start for defaults check. It existed in develop so considered trivial to check and kept. I think should I go ahead and remove this?

Reply:
Removed, but please confirm (I'm not sure what was the reason why a specific request_params == '' conditional check was added).

Reply:
Without wait time tests are failing. Hence this needs to be reverted.
Same apply to **test_default_get_audit_postgres** def

> why a specific request_params == '' conditional check was added).

Because we need wait time only once after fledge start; Not on all parameterized test data.

Reply:
What a pain, We should have wait in fledge_start fixture itself, and poll ping, if failed for x max tries * w wait time assert Failed to start/reset & start/restart

Reply:
reverted.